src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -492,27 +486,14 @@ GLHEResponseFactors::GLHEResponseFactors(EnergyPlusData &state, std::string cons
         tmpGvals.push_back(var.at("g_function_g_value").get<Real64>());
     }
 
-    bool errorsFound = false;
-
-    if (tmpLntts.size() == tmpLntts.size()) {
-        this->numGFuncPairs = static_cast<int>(tmpLntts.size());
-    } else {
-        errorsFound = true;
-        ShowSevereError(state, format("Errors found processing response factor input for Response Factor= {}", this->name));
-        ShowSevereError(state, "Uneven number of g-function pairs");
-    }
-
+    this->numGFuncPairs = static_cast<int>(tmpLntts.size());
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:497]:(style),[duplicateExpression],Same expression on both sides of '=='.